src/EditorFeatures/CSharp/CompleteStatement/CompleteStatementCommandHandler.cs

```diff
@@ -169,10 +169,10 @@ private static int GetSemicolonLocation(SyntaxNode root, SyntaxNode currentNode,
         private static bool CaretIsInForStatementCondition(int caretPosition, ForStatementSyntax forStatementSyntax)
             // If condition is null and caret is in the condition section, as in `for ( ; $$; )`, 
             // we will have bailed earlier due to not being inside supported delimiters
-            => forStatementSyntax.Condition == null ?
-                false :
-                caretPosition > forStatementSyntax.Condition.SpanStart &&
-                caretPosition < forStatementSyntax.Condition.Span.End;
+            => forStatementSyntax.Condition == null
+                ? false
+                : caretPosition > forStatementSyntax.Condition.SpanStart &&
+                  caretPosition < forStatementSyntax.Condition.Span.End;
```

Comment:
📝 This change can be reverted (this file is out of scope for the current PR)

Reply:
@sharwell i was getting a warning that the formatting here was incorrect.  I like having 0 warnings locally.  So i fixed it up.  Otherwise i think you can just get warning fatigue.